### DISCUSSION TITLE
Bug or feature - AP steers the plane off the runway

### DISCUSSION THREAD

**Discussion by uzytkownik** [flybywiresim/aircraft#4604] [Q&A] [answered]
When I update FBW to new development version after ILS landing plane is steered off the RWY during rollout unless I turn AP off. I suspect it is an issue but it may be expected behaviour.

**Comment by ExampleWasTaken** [comment, marked as answer]
That's because the ILS is badly modeled when on ground. This is something on Asobo's side and can't be fixed by us.
